src/App.jsx

```diff
@@ -1,21 +1,14 @@
 import './App.css';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
```

Comment:
Does ESLint made it to be nit?

Reply:
I think this one was just a merge conflict I needed to fix.